src/install/PackageManager/PackageManagerEnqueue.zig

```diff
@@ -450,6 +450,11 @@ pub fn enqueueDependencyWithMainAndSuccessFn(
 
     var name = dependency.realname();
 
+    // For tarballs with empty package_name, use the dependency name instead
+    if (dependency.version.tag == .tarball and name.isEmpty()) {
+        name = dependency.name;
+    }
```

Comment:
not sure how this was working before. the comment is not accurate, realname for tarballs will always return an empty string.

Reply:
I think for remote URLs when multiple are installed 

It wasn't